custom_components/spotcast/spotify/account.py

```diff
@@ -952,6 +948,36 @@ async def async_generic_playlists(
         return await self.hass.async_add_executor_job(
             partial(self._internal_cont._get, url, None, **params)
         )
+    
+    async def async_search(
```

Comment:
THere is already an async_search method. This method is generalised for any type of search. But you would have to build a SearchQuery object to utilize it

Reply:
Fixed.